_config.yml

```diff
@@ -1 +1,2 @@
-baseurl: ""
\ No newline at end of file
+baseurl: ""
+theme: jekyll-theme-cayman
```

Comment:
This line is not needed.

Reply:
Will I have to do anything to correct it? If yes, what?